src/implementations/vanilla/components/content-item/content-item-print.scss

```diff
@@ -0,0 +1,11 @@
+/**
+ * Content item
+ * @define content-item
+ */
+
+@use 'sass:map';
+@use '@ecl/theme-dev/theme';
+
```

Comment:
the date block on the print event variant is broken

Reply:
some spacing remain missing for the date block..
<img width="306" alt="Screenshot 2021-12-09 at 11 16 05" src="https://user-images.githubusercontent.com/1289480/145378456-532ad783-cbe1-4c17-b898-3cec6b2df38b.png">